test/setup/main.tf

```diff
@@ -16,7 +16,7 @@
 
 module "project" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 3.0"
+  version = "~> 9.2.0"
```

Comment:
The tests failed due to a provider version conflict.

https://github.com/terraform-google-modules/terraform-google-healthcare/blob/master/test/setup/versions.tf needs to be updated. You can use 3.46.0 as https://github.com/terraform-google-modules/terraform-google-group/blob/main/test/setup/versions.tf

Reply:
Done.